recipes/urchin/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "urchin" %}
+{% set version = "0.0.27" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/urchin-{{ version }}.tar.gz
+  sha256: bda308ed7d2b80eb1e097dc3963fabe9e00a6cbd89a1f6be6f063c2a065d3671
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
```

Comment:
You need min Python version here and below.

Reply:
I wonder the linter did not complained, then I realized that we python twice in host and in run, thanks for noticing!

Reply:
Thanks, done in https://github.com/conda-forge/staged-recipes/pull/22592/commits/8f064f61124c5ebc09597c1c8a75dac366cddfd4 .